jarviscli/plugins/speedtest.py

```diff
@@ -36,8 +36,10 @@ def stop(self):
 @plugin(network=True)
 def speedtest(jarvis, s):
     """Runs a speedtest on your internet connection"""
-
-    res = st.Speedtest()
+    try:
+        res = st.Speedtest()
+    except:
```

Comment:
As described by this code review tool:
https://app.codacy.com/app/StefanosChaliasos/Jarvis/pullRequest?prid=2920527
except-ing all kind of exceptions is usually not best idea, because you never know: If "You are not connected to Internet" shows up; is it really because of a connection error or because of another error we didn't think off?

Of course this is a really **small** issue, but nevertheless would recommend changing this line to (you should be able to apply with one click):

```suggestion
    except st.ConfigRetrievalError:
```

In addition, thanks again for you contribution!

Reply:
I was having some network problems the day I was testing it, and the specific exception didn't seem to work. It seems to be working fine now, so I will apply your suggestions.